app/plugins.js

```diff
@@ -241,7 +241,7 @@ function install(fn) {
       posix: 'npm prune && npm install --production'
     };
     // determine the shell we're running in
-    const whichShell = shell.match(/fish/) ? 'fish' : 'posix';
+    const whichShell = cfgShell.match(/fish/) ? 'fish' : 'posix';
```

Comment:
Sure we don't want to check `shell` as a fallback to `cfgShell`?  🤔  Have not had time to check the full code, but tested this branch with fish and it seems to work as expected